internal/cmd/db_import.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	dbCmd.AddCommand(importCmd)
+}
+
+var importCmd = &cobra.Command{
+	Use:               "import [filename]",
+	Short:             "Import a SQLite database file to Turso.",
+	Args:              cobra.MaximumNArgs(1),
+	ValidArgsFunction: noFilesArg,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+		filename := args[0]
+		fromFileFlag = filename
+		name := sanitizeDatabaseName(filename)
+		return CreateDatabase(name)
+	},
+}
+
+// Sanitize a SQLite database filename to be used as a Turso database name.
+func sanitizeDatabaseName(filename string) string {
+	return strings.ReplaceAll(filename, ".db", "")
+}
```

Comment:
we need to remove `/` as well if someone gives a full path? this also could fail if the name has special characters

we could also check in local cache if a db by this name exists already. if it does, we could append some random characters